.travis.yml

```diff
@@ -109,6 +111,7 @@ matrix:
         - os: linux
           stage: Initial tests
           env: CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES NUMPY_VERSION=1.12
+          compiler: clang
```

Comment:
Any suggestion where it would be more appropriate? I just put it in the initial tests so I had the result of the clang build earlier. 😄

Reply:
The more I think about it the more I like it in the initial stage. It shouldn't be with `coverage` (because that builds without optimizations) and it shouldn't be with `remote-data` because they fail (much) too often currently. And clang is used often, so failing to work with clang probably should be considered a major issue (just like failing to build with VC or gcc, right?)

Reply:
@bsipocz  Should I revert that clang-change again?

Reply:
But the osx build also use clang isnt't it ?

Reply:
well, it could stay here if you think it's more useful to get the result of that earlier than of the default gcc.

@saimn - yes, but we don't run osx any more but just from cron.

Reply:
@bsipocz - oh thanks, I thought it was not done yet.

Reply:
It's not "more useful". It's just as useful as gcc (in my opinion). 

What about if I just leave it here and if we regret it later we'll move it somewhere else?

Reply:
👍 just as useful is in the same category as more useful, so ignore this thread